manifests/init.pp

```diff
@@ -528,6 +538,10 @@
           'scrub_args' => $scrub_args,
           'custom_sensitive_words' => $custom_sensitive_words,
         },
+        'logs_enabled' => $logs_enabled,
+        'logs_config' => {
```

Comment:
Just a note, this isn't documented in the `config_template.yaml` in the datadog-agent repo. Is that intentional or oversight?

Reply:
That's a good point, opening a PR right now, thanks